doc/getting_started.rst

```diff
@@ -31,6 +31,67 @@ standards you fancy.
 .. TODO:: Mention ``npm run build`` somehow...
 
 
+Quickstart: Adding Shoop to a new project
+-----------------------------------------
+
+**Installed apps**
+
+Shoop requires several third-party apps to be added to ``INSTALLED_APPS``,
+depending on installed shoop packages. To get a shop up and running, add
+some Shoop and dependent apps in your settings.
+
+   .. code-block:: python
```

Comment:
The `.. code-block::` should start from the left most column (i.e. remove the three spaces). Similar blocks above are indented, because they are part of a numbered list.